workflows/prognostic_scream_run/scream_run/cli.py

```diff
@@ -62,8 +69,42 @@ def write_scream_run_directory():
     subprocess.run(command, shell=True)
 
 
-# TODO: add tests for scream run as part of the docker image PR
 def scream_run():
     args = _parse_scream_run_command_args()
     scream_config = _make_scream_config(args.config)
-    scream_config.submit_scream_run()
+    scream_config.submit_scream_run(args.rebuild)
+
+
+def _parse_check_config_file_command_args():
+    parser = argparse.ArgumentParser("check_config_file_command")
+    parser.add_argument(
+        "input_string",
+        help="A local path or a URI to scream yaml file, \
+            or a string of the config itself.",
+    )
+    parser.add_argument(
+        "output_config", help="Output config file name. Must be a local path.",
+    )
+    parser.add_argument(
+        "--precompiled_case", help="Whether it uses a pre-compiled case", default=True,
+    )
+    return parser.parse_args()
+
+
+def prepare_scream_config():
```

Comment:
Docstrings on the scream_run CLI would be very helpful. Getting things to run seems like it takes some additional steps beyond just what eg "write run directory" might suggest, so being clear about what needs to happen would be good.